src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,62 @@
+package lotto.controller;
+
+import java.util.List;
+import lotto.domain.Lotto;
+import lotto.domain.LottoEarningRate;
+import lotto.domain.LottoNumber;
+import lotto.domain.LottoStatistics;
+import lotto.domain.Lottos;
+import lotto.domain.WinningNumbers;
+import lotto.util.Util;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+import lotto.view.ResultView;
+
+public class LottoController {
+
+    private int lottoPrice;
+    private List<Lotto> lottoLists;
+    private Lottos lottos;
+    private String winningNumber;
+    private WinningNumbers winningNumbers;
+    private LottoNumber bonusBall;
+
+    public void start() {
+        lottoPriceProcess();
+        lottoListsProcess();
+        lottoCountProcess();
+        winningNumberProcess();
+        statisticsProcess();
+    }
+
+    private void lottoPriceProcess() {
+        OutputView.printRequestLottoPrice();
+        lottoPrice = InputView.readPrice();
+    }
+
+    private void lottoListsProcess() {
+        lottos = new Lottos(lottoPrice);
+        lottoLists = lottos.getLottoLists();
+        ResultView.printLottoNumbers(lottoLists);
```

Comment:
아래와 같이 변경할 수 있을 것 같아요. 👀 
```suggestion
        lottos = new Lottos(lottoPrice);
        ResultView.printLottoNumbers(lottos.getLottoLists());
```

Reply:
변경하였습니다!